### DISCUSSION TITLE
repository

### DISCUSSION THREAD

**Discussion by transadd** [SpiralLinux/SpiralLinux-project#144] [Q&A] [answered]
Hi
So far very good.
Before I get into Spiral Linux and start downloading and installing the programs I require, some of which are not in Debian.
Can I add deb-multimedia to the repository list or will this cause problems.
John

**Comment by geckolinux** [comment]
Hi John,
Glad it's working for you. I believe that it shouldn't cause any issues, but I've never tried it. Please report back if it works.
Thanks!

**Comment by CicadaSeventeen** [comment]
SpiralLinux does not use any own repository, so unlike using arch AUR on manjaro can make problems. Using debian repository on Spiral is OK in theory.

**Comment by transadd** [comment, marked as answer]
Hi

Thanks for your reply.
I  had been using Gecko for some time and had become accustomed to 
finding most things I needed especially with Packman enabled.
Having tried Spiral without modification using a spare disk on a caddy I 
decided that I should install on my main disk alongside my other 
distros.
For reference I have a 15 year old 64bit system and am limited to Xfce 
which I am using on Spiral  or LXQt which I used on Gecko.
As I said before so far very good and I will now probably use it as my 
main distro.
I have added deb-multimedia to my source list and have installed the 
program (DvdStyler) that was not on Debian and the program works ok.
I disable the repository when running update except when updating 
DvdStyler just in case it causes problems.
I will let you know how it goes
Thanks for your time stay safe
John


------ Original Message ------
From: "GeckoLinux" ***@***.***>
To: "SpiralLinux/SpiralLinux-project" 
***@***.***>
Cc: "transadd" ***@***.***>; "Author" 
***@***.***>
Sent: Saturday, 8 Oct, 22 At 19:38
Subject: Re: [SpiralLinux/SpiralLinux-project] repository (Discussion 
#144)


Hi John,
  Glad it's working for you. I believe that it shouldn't cause any 
issues, but I've never tried it. Please report back if it works.
  Thanks!
—
Reply to this email directly, view it on GitHub 
<https://github.com/orgs/SpiralLinux/discussions/144#discussioncomment-3830073> 
, or unsubscribe 
<https://github.com/notifications/unsubscribe-auth/AWSLWFYADSBVHCVGRETQ7ELWCG5QTANCNFSM6AAAAAARAKRN3E> 
.
You are receiving this because you authored the thread.Message ID: 
***@***.***>

**Reply by geckolinux** [reply]
Thanks for confirming!

**Comment by transadd** [comment]
Hi

It was suggested by GeckoLinux that it should work,so I have added 
deb-multimedia to my source list and enabled the repository to obtain 
the program I require.
At the moment there are no problems, time will tell if the is what I 
require and that it will not interfere with the system.
Regards stay safe
John




------ Original Message ------
From: "Cicada17" ***@***.***>
To: "SpiralLinux/SpiralLinux-project" 
***@***.***>
Cc: "transadd" ***@***.***>; "Author" 
***@***.***>
Sent: Saturday, 8 Oct, 22 At 22:32
Subject: Re: [SpiralLinux/SpiralLinux-project] repository (Discussion 
#144)


SpiralLinux does not use any own repository, so unlike using arch AUR on 
manjaro can make problems. Using debian repository on Spiral is OK in 
theory.
—
Reply to this email directly, view it on GitHub 
<https://github.com/orgs/SpiralLinux/discussions/144#discussioncomment-3830654> 
, or unsubscribe 
<https://github.com/notifications/unsubscribe-auth/AWSLWFYDL5QFZ72UG6MN6UTWCHR5PANCNFSM6AAAAAARAKRN3E> 
.
You are receiving this because you authored the thread.Message ID: 
***@***.***>
